go/vt/vtgate/evalengine/fn_compare.go

```diff
@@ -22,6 +22,7 @@ import (
 	"vitess.io/vitess/go/mysql/collations"
 	"vitess.io/vitess/go/mysql/collations/charset"
 	"vitess.io/vitess/go/mysql/collations/colldata"
+	datetime2 "vitess.io/vitess/go/mysql/datetime"
```

Comment:
nitpick: another solution would be to use the variable name `datetimes` in the functions, and keep import named `datetime`